doc/api/crypto.md

```diff
@@ -1847,7 +1847,7 @@ added: v7.10.0
 changes:
   - version: v9.0.0
     pr-url: https://github.com/nodejs/node/pull/15231
-    description: The `buffer` argument may be any ArrayBufferView
+    description: The `buffer` argument may be any `ArrayBufferView`
```

Comment:
I know I probably added this myself, but I now actually prefer `ArrayBufferView` to not be mentioned at all in the documentation, as it's not a JavaScript type ([only a Web IDL one](https://developer.mozilla.org/en-US/docs/Web/API/ArrayBufferView)). I'd say something like "may be any `TypedArray` or `DataView`".